cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/exception/ClusterManagerCheckedException.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.cluster.exception;
+
+public class ClusterManagerCheckedException extends Exception {
```

Comment:
Please remove checked word. If you named it because of ClusterManagerException in periscope, then as I see that Exception is not used, so we can delete it.

Reply:
Removed the Exception altogether. Not used yet.